src/models/voice_processing/whisper_models.py

```
@@ -0,0 +1,100 @@
+                    logger.warning(
+                        "Detected corrupted or missing model files in cache. "
+                        "Clearing cache directory: %s", cache_dir
+                    )
+                    shutil.rmtree(cache_dir)
+                    os.makedirs(cache_dir, exist_ok=True)
+                self.model = whisper.load_model(
+                    self.config.model_size,
+                    device=self.device,
+                    download_root=cache_dir
+                )
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)